metalearn/metafeatures/base.py

```diff
@@ -78,7 +78,7 @@ class MetafeatureComputer(ResourceComputer):
 
     def __init__(
         self, computer: Callable, returns: List[str], problem_type: ProblemType, groups: List[MetafeatureGroup],
-        argmap: Optional[Dict[str,str]] = None
+        argmap: Optional[Dict[str,Any]] = None
```

Comment:
Is the something more specific than `Any` we can use?

Reply:
`ResourceComputer` currently does the same thing. I’ll have to double check but I think we could safely change it to `Union[str, int]` since those are the only types currently being passed in, since all that’s been passed in are random seed values and the names of cached computations (resources).

Reply:
Thanks. I think this will help us when we have to perform any maintenance on the code.